app/shared/types/commonTypes.d.ts

```diff
@@ -71,7 +71,7 @@ export interface MetadataObjectSchema {
   provenance?: '' | 'BULK_ACCEPT';
   inheritedValue?: {
     [k: string]: unknown | undefined;
-  };
+  }[];
```

Comment:
@elreplicante this type file is auto-generated from a json schema definition https://github.com/huridocs/uwazi/blob/1ffb53f40def29a8865b598e75cd8addb667b5f6/app/shared/types/commonSchemas.ts#L105, you will need to change that file and then run yarn emit-types to update this file.